src/pantry.js

```diff
@@ -73,6 +75,9 @@ class Pantry {
       })
       .catch(error => console.log(error.message))
     })
+    $('.ingredients-confirmation').text(`You have all the ingredients needed for this recipe!`);
+    $('.ingredients-cost').text('');
+    $('li.tempIng').remove();
```

Comment:
This is a good addition to update the ingredients needed after the update of the user's pantry.